### ISSUE TITLE
conda env AttributeError importlib_resources has no attirbute "files"

### ISSUE THREAD

**Issue by moorepants** [conda/conda#11270]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help
- [X] I looked at the [docs](https://docs.conda.io/projects/conda/en/latest/user-guide/index.html) and didn't see anything to help

### What happened?

```
$ conda env list
Traceback (most recent call last):
  File "/home/moorepants/miniconda/bin/conda-env", line 6, in <module>
    from conda_env.cli.main import main
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/conda_env/cli/main.py", line 44, in <module>
    from . import main_create
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/conda_env/cli/main_create.py", line 19, in <module>
    from .. import exceptions, specs
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/conda_env/specs/__init__.py", line 9, in <module>
    from .notebook import NotebookSpec
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/conda_env/specs/notebook.py", line 5, in <module>
    import nbformat
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/nbformat/__init__.py", line 32, in <module>
    from .validator import validate, ValidationError
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/nbformat/validator.py", line 12, in <module>
    from .json_compat import get_current_validator, ValidationError
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/nbformat/json_compat.py", line 10, in <module>
    import jsonschema
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/jsonschema/__init__.py", line 29, in <module>
    from jsonschema.protocols import Validator
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/jsonschema/protocols.py", line 33, in <module>
    from jsonschema.validators import RefResolver
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/jsonschema/validators.py", line 356, in <module>
    meta_schema=_utils.load_schema("draft3"),
  File "/home/moorepants/miniconda/lib/python3.7/site-packages/jsonschema/_utils.py", line 60, in load_schema
    path = resources.files(__package__).joinpath(f"schemas/{name}.json")
AttributeError: module 'importlib_resources' has no attribute 'files'
```

All `conda env` commands are doing this.

### Conda info

<details>
<summary><code>conda info</code></summary>

```shell
$ conda info

     active environment : None
       user config file : /home/moorepants/.condarc
 populated config files : /home/moorepants/.condarc
          conda version : 4.11.0
    conda-build version : 3.21.8
         python version : 3.7.12.final.0
       virtual packages : __linux=5.4.0=0
                          __glibc=2.31=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/moorepants/miniconda  (writable)
      conda av data dir : /home/moorepants/miniconda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/pydy/linux-64
                          https://conda.anaconda.org/pydy/noarch
                          https://conda.anaconda.org/moorepants/linux-64
                          https://conda.anaconda.org/moorepants/noarch
          package cache : /home/moorepants/miniconda/pkgs
                          /home/moorepants/.conda/pkgs
       envs directories : /home/moorepants/miniconda/envs
                          /home/moorepants/.conda/envs
               platform : linux-64
             user-agent : conda/4.11.0 requests/2.27.1 CPython/3.7.12 Linux/5.4.0-100-generic ubuntu/20.04.4 glibc/2.31
                UID:GID : 1000:1000
             netrc file : /home/moorepants/.netrc
           offline mode : False
```
</details>

### Conda config

<details>
<summary><code>conda config</code></summary>

```shell
$ conda config --show-sources
==> /home/moorepants/.condarc <==
path_conflict: warn
ssl_verify: True
channel_priority: strict
channels:
  - conda-forge
  - defaults
  - pydy
  - moorepants
always_yes: True
conda-build:
  root-dir: /tmp/conda-builds
```

</details>

### Conda list

<details>
<summary><code>conda list</code></summary>

```shell
$ conda list --show-channel-urls
# packages in environment at /home/moorepants/miniconda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
alsa-lib                  1.2.3                h516909a_0    conda-forge
ampl-mp                   3.1.0             h2cc385e_1006    conda-forge
anyio                     3.5.0            py37h89c1867_0    conda-forge
argcomplete               2.0.0              pyhd8ed1ab_0    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0           py37h5e8e339_1    conda-forge
attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.10.0             pyha770c72_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
blinker                   1.4                        py_1    conda-forge
blosc                     1.21.0               h9c3ff4c_0    conda-forge
boto                      2.49.0                     py_0    conda-forge
brotli                    1.0.9                h7f98852_6    conda-forge
brotli-bin                1.0.9                h7f98852_6    conda-forge
brotli-python             1.0.9            py37hcd2ae1e_6    conda-forge
brotlipy                  0.7.0           py37h5e8e339_1003    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
cairo                     1.16.0            ha00ac49_1009    conda-forge
certifi                   2021.10.8        py37h89c1867_1    conda-forge
cffi                      1.15.0           py37h036bc23_0    conda-forge
chardet                   4.0.0            py37h89c1867_2    conda-forge
charset-normalizer        2.0.12             pyhd8ed1ab_0    conda-forge
click                     8.0.4            py37h89c1867_0    conda-forge
cmarkgfm                  0.7.0            py37h5e8e339_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
commonmark                0.9.1                      py_0    conda-forge
conda                     4.11.0           py37h89c1867_0    conda-forge
conda-build               3.21.8           py37h89c1867_0    conda-forge
conda-package-handling    1.7.3            py37h5e8e339_1    conda-forge
cryptography              36.0.1           py37hf1a17b8_0    conda-forge
css-html-js-minify        2.5.5            py37h89c1867_5    conda-forge
curl                      7.81.0               h2574ce0_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cyipopt                   1.1.0            py37h4f1d67c_1    conda-forge
cython                    0.29.28          py37hcd2ae1e_0    conda-forge
dash                      2.2.0              pyhd8ed1ab_0    conda-forge
dash-bootstrap-components 1.0.3              pyhd8ed1ab_0    conda-forge
dataclasses               0.8                pyhc8e2a94_3    conda-forge
dbus                      1.13.6               h5008d03_3    conda-forge
debugpy                   1.5.1            py37hcd2ae1e_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
derivative                0.3.2              pyhd8ed1ab_0    conda-forge
docutils                  0.16             py37h89c1867_3    conda-forge
dynamicisttoolkit         0.5.3              pyh9f0ad1d_3    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
expat                     2.4.6                h27087fc_0    conda-forge
fastcache                 1.1.0            py37h5e8e339_3    conda-forge
feedgenerator             2.0.0              pyhd8ed1ab_0    conda-forge
filelock                  3.6.0              pyhd8ed1ab_0    conda-forge
flask                     1.1.2              pyh9f0ad1d_0    conda-forge
flask-compress            1.10.1             pyhd8ed1ab_0    conda-forge
flit-core                 3.7.1              pyhd8ed1ab_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.96              ha180cfb_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.29.1           py37h5e8e339_0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
future                    0.18.2           py37h89c1867_4    conda-forge
gettext                   0.19.8.1          h73d1719_1008    conda-forge
ghp-import                2.0.2              pyhd8ed1ab_0    conda-forge
giflib                    5.2.1                h36c2ea0_2    conda-forge
git                       2.35.0          pl5321hc30692c_0    conda-forge
glob2                     0.7                        py_0    conda-forge
gmp                       6.2.1                h58526e2_0    conda-forge
gmpy2                     2.1.2            py37h025e8b9_0    conda-forge
go2                       2.20170703       py37h89c1867_3    conda-forge
grayskull                 0.9.1              pyhd8ed1ab_0    conda-forge
gst-plugins-base          1.18.5               hf529b03_3    conda-forge
gstreamer                 1.18.5               h9f60fe5_3    conda-forge
hdf5                      1.12.1          nompi_h2750804_103    conda-forge
icu                       69.1                 h9c3ff4c_0    conda-forge
idna                      3.3                pyhd8ed1ab_0    conda-forge
imagesize                 1.3.0              pyhd8ed1ab_0    conda-forge
importlib-metadata        4.11.1           py37h89c1867_0    conda-forge
importlib_metadata        4.11.1               hd8ed1ab_0    conda-forge
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipopt                     3.14.4               h7ede334_0    conda-forge
ipykernel                 6.9.1            py37h6531663_0    conda-forge
ipympl                    0.8.8              pyhd8ed1ab_0    conda-forge
ipyparallel               8.2.0              pyhd8ed1ab_0    conda-forge
ipython                   7.32.0           py37h89c1867_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.5              pyhd8ed1ab_0    conda-forge
itsdangerous              2.1.0              pyhd8ed1ab_0    conda-forge
jbig                      2.1               h7f98852_2003    conda-forge
jedi                      0.18.1           py37h89c1867_0    conda-forge
jeepney                   0.7.1              pyhd8ed1ab_0    conda-forge
jinja2                    2.11.3             pyh44b312d_0    conda-forge
joblib                    1.1.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   h7f98852_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsonpickle                2.1.0              pyhd8ed1ab_0    conda-forge
jsonschema                4.4.0              pyhd8ed1ab_0    conda-forge
jupyter                   1.0.0            py37h89c1867_7    conda-forge
jupyter-sphinx            0.3.2              pyhd8ed1ab_1    conda-forge
jupyter_client            7.1.2              pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.2            py37h89c1867_0    conda-forge
jupyter_server            1.13.5             pyhd8ed1ab_1    conda-forge
jupyterlab                3.2.9              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         2.10.3             pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        1.0.2              pyhd8ed1ab_0    conda-forge
keyring                   23.4.0           py37h89c1867_2    conda-forge
kiwisolver                1.3.2            py37h2527ec5_1    conda-forge
krb5                      1.19.2               hcc1bbae_3    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
lerc                      3.0                  h9c3ff4c_0    conda-forge
libarchive                3.5.2                hccf745f_1    conda-forge
libblas                   3.9.0           13_linux64_openblas    conda-forge
libbrotlicommon           1.0.9                h7f98852_6    conda-forge
libbrotlidec              1.0.9                h7f98852_6    conda-forge
libbrotlienc              1.0.9                h7f98852_6    conda-forge
libcblas                  3.9.0           13_linux64_openblas    conda-forge
libclang                  13.0.1          default_hc23dcda_0    conda-forge
libcurl                   7.81.0               h2574ce0_0    conda-forge
libdeflate                1.10                 h7f98852_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libevent                  2.1.10               h9b69904_4    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 11.2.0              h1d223b6_12    conda-forge
libgfortran-ng            11.2.0              h69a702a_12    conda-forge
libgfortran5              11.2.0              h5c6108e_12    conda-forge
libgirepository           1.70.0               hb520f89_1    conda-forge
libglib                   2.70.2               h174f98d_4    conda-forge
libgomp                   11.2.0              h1d223b6_12    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           13_linux64_openblas    conda-forge
liblief                   0.11.5               h9c3ff4c_1    conda-forge
libllvm13                 13.0.1               hf817b99_2    conda-forge
libnghttp2                1.47.0               h727a467_0    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libogg                    1.3.4                h7f98852_1    conda-forge
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libopus                   1.3.1                h7f98852_1    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libpq                     14.2                 hd57d9b9_0    conda-forge
libsodium                 1.0.18               h36c2ea0_1    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-ng              11.2.0              he4da1e4_12    conda-forge
libtiff                   4.3.0                h542a066_3    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libuv                     1.43.0               h7f98852_0    conda-forge
libvorbis                 1.3.7                h9c3ff4c_0    conda-forge
libwebp                   1.2.2                h3452ae3_0    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxkbcommon              1.0.3                he3ba5ed_0    conda-forge
libxml2                   2.9.12               h885dcf4_1    conda-forge
libxslt                   1.1.33               h0ef7038_3    conda-forge
libzlib                   1.2.11            h36c2ea0_1013    conda-forge
linkify-it-py             1.0.3              pyhd8ed1ab_0    conda-forge
livereload                2.6.3              pyh9f0ad1d_0    conda-forge
lxml                      4.8.0            py37h77fd288_0    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
markdown-it-py            2.0.1              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.0            py37h540881e_0    conda-forge
matplotlib                3.5.1            py37h89c1867_0    conda-forge
matplotlib-base           3.5.1            py37h1058ff1_0    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mdit-py-plugins           0.3.0              pyhd8ed1ab_0    conda-forge
mdurl                     0.1.0              pyhd8ed1ab_0    conda-forge
metakernel                0.28.2             pyhd8ed1ab_0    conda-forge
metis                     5.1.0             h58526e2_1006    conda-forge
mistune                   0.8.4           py37h5e8e339_1005    conda-forge
mpc                       1.2.1                h9f54685_0    conda-forge
mpfr                      4.1.0                h9202a9a_1    conda-forge
mpmath                    1.2.1              pyhd8ed1ab_0    conda-forge
mumps-include             5.2.1               ha770c72_10    conda-forge
mumps-seq                 5.2.1               h47a8eb5_10    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
mysql-common              8.0.28               ha770c72_0    conda-forge
mysql-libs                8.0.28               hfa10184_0    conda-forge
myst-parser               0.17.0             pyhd8ed1ab_0    conda-forge
nbclassic                 0.3.5              pyhd8ed1ab_0    conda-forge
nbclient                  0.5.11             pyhd8ed1ab_0    conda-forge
nbconvert                 6.4.2            py37h89c1867_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
ncurses                   6.3                  h9c3ff4c_0    conda-forge
nest-asyncio              1.5.4              pyhd8ed1ab_0    conda-forge
nodejs                    17.4.0               h8ca31f7_0    conda-forge
nomkl                     1.0                  h5ca1d4c_0    conda-forge
notebook                  6.4.8              pyha770c72_0    conda-forge
nspr                      4.32                 h9c3ff4c_1    conda-forge
nss                       3.74                 hb5efdd6_0    conda-forge
numexpr                   2.8.0           py37hfe5f03c_101    conda-forge
numpy                     1.21.5           py37hf2998dd_0    conda-forge
oct2py                    5.4.3              pyh8a188c0_0    conda-forge
octave_kernel             0.34.0             pyhd8ed1ab_0    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openssl                   1.1.1l               h7f98852_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.3.5            py37he8f5f7f_0    conda-forge
pandoc                    2.17.1.1             ha770c72_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parsimonious              0.8.1                      py_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
patchelf                  0.14.5               h58526e2_0    conda-forge
patsy                     0.5.2              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pcre2                     10.37                h032f7d1_0    conda-forge
pelican                   4.7.2              pyhd8ed1ab_0    conda-forge
perl                      5.32.1          2_h7f98852_perl5    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.0.1            py37h44f0d7a_2    conda-forge
pip                       22.0.3             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h36c2ea0_0    conda-forge
pkginfo                   1.8.2              pyhd8ed1ab_0    conda-forge
plotly                    5.6.0              pyhd8ed1ab_0    conda-forge
pluggy                    1.0.0            py37h89c1867_2    conda-forge
portalocker               2.4.0            py37h89c1867_0    conda-forge
progressbar2              4.0.0              pyhd8ed1ab_0    conda-forge
prometheus_client         0.13.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.27             pyha770c72_0    conda-forge
prompt_toolkit            3.0.27               hd8ed1ab_0    conda-forge
psutil                    5.9.0            py37h5e8e339_0    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
py                        1.11.0             pyh6c4a22f_0    conda-forge
py-lief                   0.11.5           py37hcd2ae1e_1    conda-forge
pycairo                   1.20.1           py37hfff247e_1    conda-forge
pycosat                   0.6.3           py37h5e8e339_1009    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycvodes                  0.14.1           py37h6fe8eb3_0    conda-forge
pydata-sphinx-theme       0.7.2              pyhd8ed1ab_0    conda-forge
pydy                      0.6.0            py37h89c1867_1    conda-forge
pyfiglet                  0.8.post1                  py_0    conda-forge
pygments                  2.11.2             pyhd8ed1ab_0    conda-forge
pygobject                 3.42.0           py37hc53cd28_1    conda-forge
pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.7              pyhd8ed1ab_0    conda-forge
pyqt                      5.12.3           py37h89c1867_8    conda-forge
pyqt-impl                 5.12.3           py37hac37412_8    conda-forge
pyqt5-sip                 4.19.18          py37hcd2ae1e_8    conda-forge
pyqtchart                 5.12             py37he336c9b_8    conda-forge
pyqtwebengine             5.12.1           py37he336c9b_8    conda-forge
pyrsistent                0.18.1           py37h5e8e339_0    conda-forge
pysocks                   1.7.1            py37h89c1867_4    conda-forge
pytables                  3.7.0            py37h5dea08b_0    conda-forge
pytest                    7.0.1            py37h89c1867_0    conda-forge
python                    3.7.12          hb7a2778_100_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-libarchive-c       4.0              py37h89c1867_0    conda-forge
python-slugify            6.1.0              pyhd8ed1ab_0    conda-forge
python-utils              3.1.0              pyhd8ed1ab_0    conda-forge
python_abi                3.7                     2_cp37m    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pyyaml                    6.0              py37h5e8e339_3    conda-forge
pyzmq                     22.3.0           py37h336d617_1    conda-forge
qt                        5.12.9               ha98a1a1_5    conda-forge
qtconsole                 5.2.2              pyhd8ed1ab_1    conda-forge
qtconsole-base            5.2.2              pyhd8ed1ab_1    conda-forge
qtpy                      2.0.1              pyhd8ed1ab_0    conda-forge
rapidfuzz                 1.9.1            py37he8f5f7f_0    conda-forge
readline                  8.1                  h46c0cb4_0    conda-forge
readme_renderer           27.0               pyh9f0ad1d_0    conda-forge
requests                  2.27.1             pyhd8ed1ab_0    conda-forge
requests-toolbelt         0.9.1                      py_0    conda-forge
resonance                 0.22.0           py37h89c1867_3    conda-forge
rfc3986                   2.0.0              pyhd8ed1ab_0    conda-forge
rich                      11.2.0             pyhd8ed1ab_0    conda-forge
ripgrep                   13.0.0               habb4d0f_1    conda-forge
ruamel.yaml               0.17.21          py37h5e8e339_0    conda-forge
ruamel.yaml.clib          0.2.6            py37h5e8e339_0    conda-forge
ruamel.yaml.jinja2        0.2.4                      py_1    conda-forge
ruamel_yaml               0.15.80         py37h5e8e339_1006    conda-forge
scikit-learn              1.0.2            py37hf9e9bfc_0    conda-forge
scikits.odes              2.6.1            py37he90e19c_3    conda-forge
scipy                     1.7.3            py37hf2a6cf1_0    conda-forge
scotch                    6.0.9                h3858553_1    conda-forge
seaborn                   0.11.2               hd8ed1ab_0    conda-forge
seaborn-base              0.11.2             pyhd8ed1ab_0    conda-forge
secretstorage             3.3.1            py37h89c1867_1    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                60.9.3           py37h89c1867_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
slugify                   0.0.1                      py_2    conda-forge
sniffio                   1.2.0            py37h89c1867_2    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3.1              pyhd8ed1ab_0    conda-forge
sphinx                    4.4.0              pyh6c4a22f_1    conda-forge
sphinx-autobuild          2021.3.14          pyhd8ed1ab_0    conda-forge
sphinx-book-theme         0.2.0              pyhd8ed1ab_1    conda-forge
sphinx-copybutton         0.5.0              pyhd8ed1ab_0    conda-forge
sphinx-js                 3.1.2              pyhd8ed1ab_0    conda-forge
sphinx-material           0.0.35             pyhd8ed1ab_0    conda-forge
sphinx-math-dollar        1.2                pyh9f0ad1d_0    conda-forge
sphinx-reredirects        0.0.1              pyhd8ed1ab_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_1    conda-forge
sqlite                    3.37.0               h9cd32fc_0    conda-forge
statsmodels               0.13.2           py37hb1e94ed_0    conda-forge
stdlib-list               0.7.0                      py_2    conda-forge
suitesparse               5.10.1               h9e50725_1    conda-forge
sundials                  5.6.1                h800c78e_0    conda-forge
sympy                     1.9              py37h89c1867_1    conda-forge
tbb                       2021.5.0             h4bd325d_0    conda-forge
tenacity                  8.0.1              pyhd8ed1ab_0    conda-forge
termcolor                 1.1.0                      py_2    conda-forge
terminado                 0.13.1           py37h89c1867_0    conda-forge
testpath                  0.6.0              pyhd8ed1ab_0    conda-forge
text-unidecode            1.3                        py_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tk                        8.6.12               h27826a3_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tornado                   6.1              py37h5e8e339_2    conda-forge
tqdm                      4.62.3             pyhd8ed1ab_0    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
tree                      2.0.0                h7f98852_0    conda-forge
twine                     3.8.0              pyhd8ed1ab_0    conda-forge
typing-extensions         4.1.1                hd8ed1ab_0    conda-forge
typing_extensions         4.1.1              pyha770c72_0    conda-forge
uc-micro-py               1.0.1              pyhd8ed1ab_0    conda-forge
uncertainties             3.1.6              pyhd8ed1ab_0    conda-forge
unicodedata2              14.0.0           py37h5e8e339_0    conda-forge
unidecode                 1.3.3              pyhd8ed1ab_0    conda-forge
unixodbc                  2.3.9                hb166930_0    conda-forge
urllib3                   1.26.8             pyhd8ed1ab_1    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.3.1              pyhd8ed1ab_0    conda-forge
werkzeug                  2.0.3              pyhd8ed1ab_1    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
widgetsnbextension        3.5.2            py37h89c1867_1    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h7f98852_0    conda-forge
xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
yeadon                    1.3.0           py37h89c1867_1004    conda-forge
youtube-dl                2021.12.17         pyhd8ed1ab_0    conda-forge
zeromq                    4.3.4                h9c3ff4c_1    conda-forge
zipp                      3.7.0              pyhd8ed1ab_1    conda-forge
zlib                      1.2.11            h36c2ea0_1013    conda-forge
zstd                      1.5.2                ha95c52a_0    conda-forge
```
</details>

### Additional Context

I assume another package in my environment has updated and caused this.

**Comment by moorepants** [comment]
I downgraded to jsonschema 3.2.0 and no longer get the error. `conda env` commands work with jsonschema 3.2.0.

**Comment by kenodegard** [comment]
The recommended fix is to downgrade `jsonschema` as you have identified.

We also recommend not installing packages into your base environment for this very reason. It is preferable to instead create new environments (`conda create ...`) where you install the packages you wish to use. By using environments you avoid the risk of corrupting your base environment with packages that are incompatible with your `conda` installation.

**Comment by moorepants** [comment]
I do you use environments, but I've had a base environment for my default work. I've use a general base environment since conda was first released.

**Comment by moorepants** [comment]
If jsonschema 4.4.0 breaks conda, shouldn't conda have a dependency pin set as `jsonschema<4`?

**Comment by moorepants** [comment]
I've opened a fix on the conda forge feedstock: https://github.com/conda-forge/conda-feedstock/pull/160

**Comment by moorepants** [comment]
I think I've found the actual issue here: https://github.com/conda-forge/importlib_resources-feedstock/issues/46

**Comment by jezdez** [comment]
@moorepants thanks for looking into this, so in other words that is an issue with the importlib_resources feedstock in conda-forge that happened because of some name normalization?

**Comment by moorepants** [comment]
Yes, that is what it seems. For some reason I'm ending up with both `importlib_resources` and `importlib-resources` (both different version numbers) in my conda env. Still not quite sure why and I'm struggling to remove it with `conda remove importlib-resources` (conda kills itself after a time out on the env solve).

**Comment by jezdez** [comment]
> Yes, that is what it seems. For some reason I'm ending up with both `importlib_resources` and `importlib-resources` (both different version numbers) in my conda env.

Could you check with https://github.com/rvalieris/conda-tree and see if there are other packages relying on the wrong name?

> Still not quite sure why and I'm struggling to remove it with `conda remove importlib-resources` (conda kills itself after a time out on the env solve).

Ugh, sorry to hear that the solver performance is in the way here, in doubt you could try getting rid of it using mamba to clear your environment?

**Comment by moorepants** [comment]
mamba worked. I posted some details over at https://github.com/conda-forge/importlib_resources-feedstock/issues/46

**Comment by moorepants** [comment]
Here is a summary:

- I installed jupyter-book in my base environment. jupyter-book has very strict upper bound pins (see https://github.com/executablebooks/meta/issues/648). For some reason, this caused an old importlib_resources to be installed alongside the up-to-date version.
- I used mamba to remove jupyter-book and sphinx-book-theme and their associated dependencies.
- I then used mamba to remove `importlib-resources` leaving only the newer `importlib_resources` installed.
- Finally I had to `conda update --force-reinstall importlib_resources` to make this package usable (the last command left it empty of files).
- Then `conda env list` works.
